### PR TITLE
Update project docs

### PR DESCRIPTION
**Pull request by jleach**
Add or update documentation so this can be slurped in and processed by the DevHub.

### CODE REVIEW

File: .devhubignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+.git/
+/.gitignore
+/.prettierrc
+.s2i/
+.vscode/
+/Caddyfile
+/Dockerfile
+/Jenkinsfile
+/LICENSE
+README.md
```
Comment (patricksimonian) [inline .devhubignore]: you'll need to set this too `/README.md` otherwise it won't be ignored.

File: .devhubignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/devhub.md
```
Comment (patricksimonian) [inline .devhubignore]: this should be removed from the ignore file otherwise the devhub will ignore it and not generate a card for that markdown file

**Review by patricksimonian — CHANGES_REQUESTED** [review]

File: .devhubignore (hunk 1 above)
Comment (jleach) [reply .devhubignore]: Fixed.

File: .devhubignore (hunk 2 above)
Comment (jleach) [reply .devhubignore]: Fixed.

**Review by patricksimonian — APPROVED** [review]

**Pull request merged**